docs/product/resources/svg-icons/ArrowUpDownSm.svg

```diff
@@ -1 +1 @@
-<svg aria-hidden="true" class="svg-icon iconArrowUpDownSm" width="14" height="14" viewBox="0 0 14 14"><path d="M3 8h8l-4 4zm0-2h8L7 2z"/></svg>
\ No newline at end of file
+<svg aria-hidden="true" class="svg-icon iconArrowUpDownSm" width="14" height="14" viewBox="0 0 14 14"><path d="M3 8h8l-4 4zm0-2h8L7 2z"/></svg>
```

Comment:
What's going on here? Did we add a new line at the end of this file somehow?

Reply:
It was modified earlier in the PR and this reverted it back to what was in `develop`. No changes to the icon itself.